### ISSUE TITLE
Wrong import module name! it is : stempel

### ISSUE THREAD

**Issue by Manamama** [dzieciou/pystempel#31]
> Type "help", "copyright", "credits" or "license" for more information.
> >>> from pystempel import Stemmer
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
> ModuleNotFoundError: No module named 'pystempel'
> >>> 
> ~/Downloads$ pip show stempel
> WARNING: Package(s) not found: stempel
> ~/Downloads$ pip show pystempel
> Name: pystempel
> Version: 1.2.0
> Summary: Polish stemmer.
> Home-page: https://github.com/dzieciou/pystempel
> Author: Maciej Gawinecki
> Author-email: mgawinecki@gmail.com
> License: See documentationpython
> 
> >>> from pystempel import Stemmer
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
> ModuleNotFoundError: No module named 'pystempel'
> >>> 
> ~/Downloads$ pip show stempel
> WARNING: Package(s) not found: stempel
> ~/Downloads$ pip show pystempel
> Name: pystempel
> Version: 1.2.0
> Summary: Polish stemmer.
> Home-page: https://github.com/dzieciou/pystempel
> Author: Maciej Gawinecki
> Author-email: mgawinecki@gmail.com
> License: See documentation
> Requires: sortedcontainers, tqdm
> Required-by: 
> 

Only this works:
`from stempel import StempelStemmer
`

etc. 

e.g. 

```

class CallableStemmer:
    def __init__(self, stemmer):
        self.stemmer = stemmer

    def __call__(self, word):
        return self.stemmer.stem(word)

def get_stemmer(language):
    if language == 'pol':
        # Create a StempelStemmer object for Polish
        stemmer_obj = StempelStemmer.default()
    else:
        # Handle other languages here...
        raise ValueError(f'Unsupported language: {language}')

    # Wrap it in a CallableStemmer
    return CallableStemmer(stemmer_obj)

```

and then 
```

    #stemmer = Stemmer(language)
    stemmer = get_stemmer(language)

```

for the [sumy](https://github.com/miso-belica/sumy) package.

**Comment by dzieciou** [comment]
Thanks for your report. I will try to fix that as soon as find some time.

**Comment by dzieciou** [comment]
Fixed by #33. Feel free to re-open if it does not fix your issue.

**Comment by dzieciou** [comment]
If you're using release 1.2.0 version you should follow docs for this exact version on https://pypi.org/project/pystempel/.

The documentation in GitHub repo is for 2.0.0 version, not release yet.

Rolling back.
